OpenRA.Mods.Common/Traits/TransformOnCapture.cs

```diff
@@ -20,6 +20,9 @@ public class TransformOnCaptureInfo : ITraitInfo
 		public readonly int ForceHealthPercentage = 0;
 		public readonly bool SkipMakeAnims = true;
 
+		[Desc("Triggered only capturer's CaptureType contains this.")]
```

Comment:
I'm having difficulty parsing this sentence.  Typo or missing word?

Reply:
"only if Capturer's"